config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yaml

```diff
@@ -43,13 +43,6 @@ periodics:
       # don't retry conformance tests
       - name: GINKGO_TOLERATE_FLAKES
         value: "n"
-      - name: FOCUS
-        value: \[Conformance\]|\[Driver:.csi-hostpath\]
```

Comment:
This disables the storage tests that I just added in https://github.com/kubernetes/test-infra/pull/24410.

Is that intentional?

As I said in https://github.com/kubernetes/test-infra/pull/24410/files#r752422365, the config that you are changing here hasn't been used at all yet.

Reply:
It doesn't disable them; Focus is a filter/allow-list. If you don't specify it, you get all the tests.

Reply:
In this case FOCUS actually has to be . if you want all, because https://github.com/kubernetes-sigs/kind/blob/d1eecc46e30cac9d35cd32dc52677ef75ec22e18/hack/ci/e2e-k8s.sh#L225

Reply:
https://github.com/kubernetes/test-infra/pull/24410 worked as intended, recent job runs executed conformance and storage tests.

Reply:
> It doesn't disable them; Focus is a filter/allow-list. If you don't specify it, you get all the tests.

Removing the FOCUS variable causes the script to use the default, which is just `\[Conformance\]`. Therefore the change above removes the `\[Driver:.csi-hostpath\]` pattern and thus the storage tests that use that driver.